cmd/cl-adm/cmd/deploy/deploy_peer.go

```diff
@@ -84,6 +86,8 @@ func (o *PeerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.Namespace, "start-namespace", app.SystemNamespace,
```

Comment:
I'm not too fond of the use of `start` command line option. I think `autostart` or `deploy` would have been better.
Or even better - make it a separate command altogether?

Also, consider dropping the `start-` prefix from variables. There are two options:
- if start is not specified, log a warning that these are ignored; or
- save them into a configmap to be used later
If you make it a separate command, then neither option is required.